mail/rspamd/Portfile

```diff
@@ -1,79 +1,101 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; truncate-lines: t; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           cmake 1.0
+PortGroup           cmake 1.1
 PortGroup           github 1.0
 
-github.setup        vstakhov rspamd 0.8.3
+github.setup        rspamd rspamd 1.9.2
 
-version             0.8.3
-revision            3
+name                rspamd
+version             1.9.2
```

Comment:
Both the `name` and `version` are already set by the `github.setup` command. So please remove these lines.